util/include/JsonSerializer.hpp

```diff
@@ -0,0 +1,49 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
```

Comment:
I thought about it and decided to write a new one from 2 reasons -
1) not sure about the validation of the output (Andrew said to me that it's really important).
2) I cant use this library cause the library use because kvbc uses replicaimp so I cant use the opposite way.

Reply:
I think it is a good idea to reuse code. However, this serialiser is extremely simplified and that's why I didn't put it in `util`. It lacks a lot of features and is not generic enough.

Additionally, I, personally, wouldn't want us to write our own full-blown JSON library.

Therefore, my suggestion is to either:

- use an external JSON library
- put it in `util`, but don't waste effort extending it

Reply:
@Liorgal28 , as Petar said we can just move it to `util`.
@dartdart26 , since we have more and more usage of JSON, it seems reasonable to use a seasoned 3rd party.